src/interpreter-errors.ts

```diff
@@ -89,11 +94,20 @@ export class CallingNonFunctionValue extends RuntimeSourceError {
   }
 
   public explain() {
-    return `Calling non-function value ${stringify(this.callee)}`
+    return `Calling non-function value ${stringify(this.callee)}.`
   }
 
   public elaborate() {
-    return 'TODO'
+    const calleeVal = this.callee
+    const calleeStr = stringify(calleeVal)
+    const elabStr = `Because ${calleeStr} is not a function, you cannot run ${calleeStr}().`
```

Comment:
Is "you cannot run notfunction()" confusing, when maybe their code was `notfunction(1, 2, 3)`?